### PR TITLE
Added servlet to fetch hard coded association data from and display t…

### PR DESCRIPTION
**Pull request by prashantiA**
Changes:
- Associations fetched from servlet instead of hard coded into the html

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/data/AssociationData.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.ArrayList;
+
+/** Data for top negative and positive associations  */
+public final class AssociationData {
+
+  private ArrayList<String> positive;
+  private ArrayList<String> negative;
+	      
+  public AssociationData(ArrayList<String> positiveAssociations, ArrayList<String> negativeAssociations) {
```
Comment (nitinramadoss) [inline capstone/src/main/java/com/google/sps/data/AssociationData.java:11]: Ensure that you keep the column limit under 100 characters. Java has a specific guideline that recommends keeping column character limits below 100
https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

File: capstone/src/main/java/com/google/sps/servlets/AssociationServlet.java [hunk 2]
```diff
@@ -14,19 +14,30 @@
...
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+/** Servlet to fetch association data with most positive and negative associations */
+@WebServlet("/associations")
+public class AssociationServlet extends HttpServlet {
+
+  private static final String OUTPUT_TYPE = "applications/json;";
+  private static final ArrayList<String> positive = new ArrayList(Arrays.asList("test 1", "test 2", "test 3"));
+  private static final ArrayList<String> negative = new ArrayList(Arrays.asList("yeet 1", "yeet 2", "yeet 3"));
```
Comment (nitinramadoss) [inline capstone/src/main/java/com/google/sps/servlets/AssociationServlet.java:33]: yeet xD

File: capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlet;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import com.google.gson.Gson;
+import com.google.sps.servlets.AssociationServlet;
+import com.google.sps.data.AssociationData;
+import java.io.*;
```
Comment (jasperlee-google) [inline capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java:8]: Please avoid wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlet;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```
Comment (lee12jin) [inline capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java:4]: +1 to jasper comments for each of these.

File: capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import com.google.gson.Gson;
+import com.google.sps.servlets.AssociationServlet;
+import com.google.sps.data.AssociationData;
+import java.io.*;
+import java.util.Arrays;
+import java.util.ArrayList;
+import javax.servlet.http.*;
```
Comment (lee12jin) [inline capstone/src/test/java/com/google/sps/servlets/AssociationServletTest.java:11]: +1 to Jasper's comment here

**Comment by prashantiA** [comment]
Fixed the style issues, merging!

**Pull request merged**